### ISSUE TITLE
Is it intentional that the current path is prefixed with `main::` or `lib::`?

### ISSUE THREAD

**Issue by TheButlah** [Shizcow/caller_modpath#4]
Is it to disambiguate between bin and lib crates?

**Comment by Shizcow** [comment]
You are correct, this is to tell between bin and lib crates. Due to the way this crate parses the Cargo.toml file and finds the source file of the invocation, I remember baking this info in to more easily deal with the forked rustc call. Since that information was already available, I thought it useful to expose it to the user -- more info is usually better.
